astro/marble/pkg-plist

```diff
@@ -1750,16 +1981,16 @@ share/locale/zh_TW/LC_MESSAGES/plasma_wallpaper_org.kde.plasma.worldmap.mo
 %%DATADIR%%/data/svg/thenounproject/204712-hiker.svg
 %%DATADIR%%/data/svg/thenounproject/21636-gondola.svg
 %%DATADIR%%/data/svg/thenounproject/231-chair-lift.svg
-%%DATADIR%%/data/svg/thenounproject/2412-skiing-downhill.svg
-%%DATADIR%%/data/svg/thenounproject/245-ski-jump.svg
 %%DATADIR%%/data/svg/thenounproject/29366-skitour.svg
 %%DATADIR%%/data/svg/thenounproject/30231-skiing-cross-country.svg
 %%DATADIR%%/data/svg/thenounproject/365217-sled.svg
 %%DATADIR%%/data/svg/thenounproject/583813-cable-car.svg
 %%DATADIR%%/data/svg/thenounproject/61698-mountain-biking.svg
 %%DATADIR%%/data/svg/thenounproject/78374-horse-riding.svg
 %%DATADIR%%/data/svg/thenounproject/8803-platter-lift.svg
-%%DATADIR%%/data/svg/thenounproject/sports-245-ski-jump.svg
+%%DATADIR%%/data/svg/thenounproject/%%KDE_APPLICATIONS_VERSION_SHORT%%12-skiing-downhill.svg
```

Comment:
These next few plist subs with %%KDE_APPLICATIONS_VERSION_SHORT%% are incorrect. They should just be 24 and sorted back to where they were.